package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ah-codeblooded-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-scripts": "2.0.5"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "node_modules/.bin/eslint src/"
```

Comment:
I think you don't need to write the full path for the eslint...node will automatically pick up the eslint since the lint command is under scripts..you could write like this as well ```"lint": "eslint src/"```

Reply:
@imireallan , I have removed the redundant relative path that can be automatically picked by node. Thanks